packages/experimental/gem-globe/src/components/Globe.tsx

```diff
@@ -256,8 +256,8 @@ const GlobeZoomControls = ({ classNames, position = 'bottom-left', onAction }: G
 };
 
 const GlobeActionControls = ({ classNames, position = 'bottom-right', onAction }: GlobeControlsProps) => {
+  // TODO(wittjosiah): This is a hack to get the theme to work. Gem isn't getting global theme context.
```

Comment:
this comment makes more sense directly beside the component to which it is referring